tests/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ add_executable(testPowerWAF ${LIBDDWAF_SOURCE} ${LIBDDWAF_TEST_SOURCE})
 option(LIBDDWAF_TEST_COVERAGE "Do coverage if possible" OFF)
 
 set_target_properties(testPowerWAF PROPERTIES
-    CXX_STANDARD 17
+    CXX_STANDARD 20
```

Comment:
Out of curiosity since I have no much experience with c++. Is it that easy to change from on standard to another? I mean it does not have any collateral impact anywhere else or having to change code that was on 17?

Reply:
Generally, language changes to the standard are backwards compatible, STL changes on the other hand aren't as they might deprecate older methods and functions in favour of new ones. Having new classes, methods and functions can potentially cause issues if dynamically linking against a version of libstdc++ or libc++ missing these new symbols. However in this case the standard change is limited to the tests.